code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -207,7 +207,7 @@
 		and Element Cuban combined with the Compound Pete. Can hold up to \
 		300 units."
 	icon_state = "beakerbluespace"
-	materials = list(MAT_GLASS=3000)
+	materials = list(MAT_GLASS = 5000, MAT_PLASMA = 3000, MAT_DIAMOND = 2000, MAT_BLUESPACE = 1000)

```

Comment:
They have more diamond in them than they need to be made

Reply:
reeee